src/vivarium_gates_bep/model_specifications/model_spec.in

```diff
@@ -30,7 +30,7 @@ components:
 
         risks.implementations.low_birth_weight_and_short_gestation:
             - LBWSGRisk()
-            - LBWSGRiskEffect('cause.neonatal_sepsis_and_other_neonatal_infections.excess_mortality_rate')
+            # - LBWSGRiskEffect('cause.neonatal_sepsis_and_other_neonatal_infections.excess_mortality_rate')
```

Comment:
Is this intended to remain no effect?

Reply:
Hmm, I thought that cause a naming conflict but I just uncommented and ran. Fixed.